dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/dataverse/DataverseServiceIT.java

```diff
@@ -217,13 +217,13 @@ public void publishDataverse_WithIllegalCommandException() {
     public void deleteDataverse() {
         //given
         loginSessionWithSuperUser();
-        Dataverse unpublishedDataverse = dataverseDao.find(19L);
+        Dataverse unpublishedDataverse = dataverseDao.find(67L);
```

Comment:
If dataverse have publicationDate then it is not really unpublished.

Reply:
What was the problem with deleting ROOT -> unreleased dataverse?

Reply:
In other tests we've enriched previous dataverse (with contacts, files ect.) witch caused "Dataverse not empty" error. It was easier to create empty new dataverse than to change the old one for all tests.

Reply:
Ok. So this was normal situation. I think it would be better If we test that situation also.
We could create extra method `deleteDataverse_with_files()` and check if it will throw error.

Reply:
It's more general. Before delete on dataverse we check if it `hasData()`. If it does then the exception is being thrown, else we proceed with deletion. So extra test could check for expected exception for not 'empty' dataverse.

Reply:
`hasData()` in context of dataverse is equivalent to situation where dataverse have child dataverse or have some datasets